blog/src/Widget/OffsetPagination.php

```diff
@@ -78,8 +78,8 @@ protected function prepareButtons(): void
         if ($this->prepared) {
             return;
         }
-        $this->pagesCount = $this->paginator->getTotalPages();
-        $this->currentPage = $this->paginator->getCurrentPage();
+        $this->pagesCount = null!== $this->paginator ? $this->paginator->getTotalPages() : 0;
```

Comment:
```suggestion
        $this->pagesCount = $this->paginator->getTotalPages() ?? 0;
```

Could it be done shorter?

Reply:
`$this->paginator?->getTotalPages() ?? 0;`

Reply:
Psalm Level 2 Testing has brought the following to my attention: 

ERROR: MissingConstructor - src/Widget/OffsetPagination.php:18:17 - App\Widget\OffsetPagination has an uninitialized property App\Widget\OffsetPagination::$pagesCount, but no constructor (see https://psalm.dev/073)
    private int $pagesCount;


ERROR: MissingConstructor - src/Widget/OffsetPagination.php:19:17 - App\Widget\OffsetPagination has an uninitialized property App\Widget\OffsetPagination::$currentPage, but no constructor (see https://psalm.dev/073)
    private int $currentPage;


ERROR: MissingConstructor - src/Widget/OffsetPagination.php:20:19 - App\Widget\OffsetPagination has an uninitialized property App\Widget\OffsetPagination::$pages, but no constructor (see https://psalm.dev/073)
    private array $pages;


ERROR: MissingConstructor - src/Widget/OffsetPagination.php:21:18 - App\Widget\OffsetPagination has an uninitialized property App\Widget\OffsetPagination::$prepared, but no constructor (see https://psalm.dev/073)
    private bool $prepared;

If I change the code to the following:
    private int $pagesCount = 0;
    private int $currentPage = 0;
...both these errors disappear but we are left with $pages and $prepared.